README.md

```diff
@@ -37,15 +37,12 @@
 
 ## Introduction
 
-DietPi is an extremely lightweight Debian OS. With images starting at 400MB, that's 3x lighter than 'Raspbian Lite'.
-It is highly optimised for minimal CPU and RAM resource usage, ensuring your SBC always runs at its maximum potential.
-The programs use lightweight Whiptail menus. You'll spend less time staring at the command line, and more time enjoying DietPi.
+DietPi is an extremely lightweight Debian OS. With images starting at 400MB, that's 3x lighter than 'Raspbian Lite'. It is highly optimized for minimal CPU and RAM resource usage, ensuring your SBC always runs at its maximum potential. The programs use lightweight Whiptail menus. You'll spend less time staring at the command line, and more time enjoying DietPi.
 
-Use `dietpi-software` to quickly and easily install popular software that's "Ready to run" and optimised for your system. Only the software you need is installed.
-Use `dietpi-process` to control which installed software has higher or lower priority levels (nice, affinity, policy scheduler).
+Use 'dietpi-software' to quickly and easily install popular software that's "Ready to run" and optimized for your system. Only the software you need is installed. Use dietpi-process to control which installed software has higher or lower priority levels (nice, affinity, policy scheduler).
+
+'dietpi-update' automatically checks for updates and informs you when they are available. Update instantly, without having to write a new image. 'dietpi-automation' Allows you to completely automate a DietPi installation with no user input. Simply by configuring dietpi.txt before powering on.
```

Comment:
I agree with this being more readable, but I would keep the code quotations (e.g. `dietpi-software`) to underline this being the commands you can enter into the terminal.

Also I found an error our side, it should be `dietpi-process_tool` instead of just `dietpi-process`.